src/main/index.js

```diff
@@ -21,6 +21,7 @@ var windows = require('./windows')
 
 var shouldQuit = false
 var argv = sliceArgv(process.argv)
+var hidden = argv.includes('--hidden')
```

Comment:
need to leave it here so that the main windows starts hidden,
placing the hide function in the processArgv function makes the window flash for a sec.
Made the processArgv function ignore the value as a torrent id